pom.xml

```diff
@@ -529,6 +532,19 @@
                 <ranger.version>2.0.0</ranger.version>
                 <solr.version>7.7.1</solr.version>
             </properties>
+            <build>
```

Comment:
can we use `build-helper-maven-plugin` to `add-test-source` instead of excluding  them?

Reply:
I didn't test it, going to do it and report here